cypress/support/step-definitions/radio-button-steps.js

```diff
@@ -1,79 +1,54 @@
-import { fieldHelpPreview, labelByPosition, labelWidthSliderByName } from '../../locators';
+import { labelWidthSliderByName, labelByPositionNoIFrame } from '../../locators';
 import {
-  radioButtonByPosition,
   radioButtonFieldset,
   radioButtonLegend,
   radioButtonGroupNoIframe,
   radioButtonComponentByPositionNoIframe,
 } from '../../locators/radioButton/index';
 import { setSlidebar, positionOfElement } from '../helper';
-
-const INLINE = 'carbon-radio-button__help-text--inline';
-const RADIOBUTTON_INPUT_CSS = 'div:nth-child(1) input';
-
-Then('fieldHelpInline is enabled', () => {
-  fieldHelpPreview().should('have.class', INLINE);
-});
-
-Then('fieldHelpInline is disabled', () => {
-  fieldHelpPreview().should('not.have.class', INLINE);
-});
+import { FIELD_HELP_PREVIEW } from '../../locators/locators';
```

Comment:
we should avoid using imports from `locators` in `steps.` Move it to `index.js` file please.

Reply:
fixed